src/test/kotlin/eu/europa/ec/eudi/openid4vci/examples/WalletInitiatedIssuanceUsingAuthorizationFlow.kt

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (c) 2023 European Commission
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package eu.europa.ec.eudi.openid4vci.examples
+
+import eu.europa.ec.eudi.openid4vci.*
+import eu.europa.ec.eudi.openid4vci.internal.ensure
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.client.request.forms.*
+import io.ktor.http.*
+import kotlinx.coroutines.runBlocking
+import org.jsoup.Jsoup
+import org.jsoup.nodes.FormElement
+import java.net.URL
+
+private val CredentialIssuer_URL = "https://dev.issuer-backend.eudiw.dev"
+private val actingUser = ActingUser("tneal", "password")
+
+fun main(): Unit = runBlocking {
```

Comment:
Consider to remain fun main to something else let's say `runUseCase` with two input parameters

- Credential Issuer Identifier &
- List of configurations id(s) 

Then provide a `main` which calls it

In this way it will be clear what is the "knowledge"  of the wallet (to initiate the flow)

Reply:
done